front/src/components/CTF/LogoLink.vue

```diff
@@ -1,14 +1,23 @@
 <template>
   <q-btn
-    :href="ctf.ctfUrl"
+    :href="ctf.ctfUrl || null"
     target="_blank"
     class="logo text-white"
+    :class="{ 'no-hover': !ctf.ctfUrl }"
     type="a"
     :style="style"
     :icon="icon"
+    :ripple="false"
     round
   >
-    <q-tooltip>Browse CTF website</q-tooltip>
+    <q-tooltip>
+      <span v-if="ctf.ctfUrl">
+        Browse CTF website
+      </span>
+      <span v-else>
+        No CTF website set
+      </span>
```

Comment:
Depending on your decision, you can remove this or leave it here.

Reply:
I think leaving this in makes it seem more obvious why the button is unclickable, vs. a use thinking their browser/CTFNote is broken.

Reply:
> I prefer keeping the logo for consistency.

Then I totally agree that this should be shown.